atom/browser/native_window_mac.mm

```diff
@@ -1212,7 +1212,7 @@ static bool FromV8(v8::Isolate* isolate, v8::Handle<v8::Value> val,
 }
 
 void NativeWindowMac::SetVibrancy(const std::string& type) {
-  if (!(base::mac::IsOSMavericks() || base::mac::IsOSYosemiteOrLater())) return;
+  if (!base::mac::IsOSYosemiteOrLater()) return;
```

Comment:
Looks like `NSVisualEffectView` is yosemite and above, https://developer.apple.com/reference/appkit/nsvisualeffectview